toolchain/check/context.cpp

```diff
@@ -100,14 +100,7 @@ auto Context::AddPlaceholderInst(SemIR::ParseNodeAndInst parse_node_and_inst)
 
 auto Context::AddConstant(SemIR::Inst inst, bool is_symbolic)
     -> SemIR::ConstantId {
-  auto [inst_id, added] = constants().GetOrAdd(inst);
-  auto const_id = is_symbolic ? SemIR::ConstantId::ForSymbolicConstant(inst_id)
-                              : SemIR::ConstantId::ForTemplateConstant(inst_id);
-  if (added) {
-    // TODO: Should `ConstantStore` do this for us?
-    constant_values().Set(inst_id, const_id);
-  }
-
+  auto const_id = constants().GetOrAdd(inst, is_symbolic);
   CARBON_VLOG() << "AddConstantInst: " << inst << "\n";
```

Comment:
```suggestion
  CARBON_VLOG() << "AddConstant: " << inst << "\n";
```

This feels left behind when the function name changed